libvmaf/tools/cli_parse.c

```diff
@@ -78,6 +79,75 @@ static unsigned parse_unsigned(const char *const optarg, const int option,
     return res;
 }
 
+static int parse_config(VmafModelConfig **config,
```

Comment:
I think it's possible to write this in a way that it requires no extra memory allocation. The strings are part of `argc[]`, so no extra `memcpy` should be required. You can return a `VmafModelConfig` directly to avoid having to pass it as a pointer.

Reply:
Also, call this `parse_model_config()`.